app/helpers/application_helper.rb

```diff
@@ -375,4 +375,13 @@ def get_page_name
   def label_with_asterisk(label_name)
     "#{label_name} <i class='text-danger'>*</i>".html_safe
   end
+
+  # helper for rendering split chunks tags with a nonce
+  # based off of nonced_javascript_pack_tag in secure_headers
+  # see https://github.com/github/secure_headers/blob/main/lib/secure_headers/view_helper.rb
+  def nonced_javascript_pack_with_chunks_tag(*args, &block)
```

Comment:
Took me a minute to realize what this was doing - nice.

Reply:
yeah, this definitely more technical Ruby code than I usually write, but I wanted it to match the built-in helpers as close as possible